narui_widgets/src/subpass.rs

```diff
@@ -224,15 +223,17 @@ pub fn raw_blur(
                   z_index| {
                 let window = window_function(context, abs_pos, rect);
 
-                let mut set_builder = PersistentDescriptorSet::start(
-                    pipeline.layout().descriptor_set_layouts()[0].clone(),
-                );
-                set_builder
-                    .add_sampled_image(depth, sampler.clone())
-                    .unwrap()
-                    .add_sampled_image(color, sampler.clone())
-                    .unwrap();
-                let descriptor_set = set_builder.build().unwrap();
+                // dbg!(pipeline.layout().set_layouts());
+                let descriptor_set = PersistentDescriptorSet::new(
+                    pipeline.layout().set_layouts()[0].clone(),
+                    [
+                        // TODO(robin): as long as it is unused it gets optimized out??
```

Comment:
what does this mean?

Reply:
well we do not use the depth sampler in shader, but this causes `set_layouts()[0]` to not contain a entry for it, so if we add the depth sampler here it errors out. Arguably this is a vulkano bug? I did not really investigate further...